auth4genai/calling-apis/langgraph/get-started/call-others-apis-on-users-behalf.mdx

```diff
@@ -0,0 +1,140 @@
+---
+title: Calling third-party  Party APIs
+description: Learn how Auth0 for AI Agents enables AI agents to call other's APIs on the user's behalf.
+---
+
+import UsingAI from "/snippets/common/using-ai.mdx";
+import { AIContainer } from "/snippets/common/ai-container.jsx";
+import { DownloadManual } from "/snippets/common/download-manual.jsx";
+
+{/* `GettingStartedTocs` requires `CustomTocs` to be imported */}
+import { CustomTocs } from "/snippets/common/tocs.mdx";
+import { GettingStartedTocs } from "/snippets/common/getting-started-tocs.mdx";
+
+{/* `Prerequisites` requires `AccountAndAppSteps` to be imported */}
+import { AccountAndAppSteps } from "/snippets/get-started/prerequisites/account-app-steps.jsx";
+import { Prerequisites } from "/snippets/get-started/prerequisites/call-your-api.jsx";
+
+import LangChainNextjsCallOthersApi from "/snippets/get-started/langchain-next-js/call-others-api.mdx";
+import FastApiCallOthersApi from "/snippets/get-started/langchain-fastapi-py/call-others-api.mdx";
+
+Use Auth0 SDKs to fetch access tokens for social and enterprise identity providers from Auth0's [Token Vault](https://auth0.com/docs/secure/tokens/token-vault). Using these access tokens, AI agents integrated with the application can call third-party APIs to perform tasks on the user's behalf.
+
+<Card
+  title="What you’ll learn"
+>
+  - Retrieve access tokens for a Google social connection.
+  - Integrate with an AI agent to call Google APIs.
+</Card>
+
+<Prerequisites />
```

Comment:
a lot of these Prerequisites have recently been adjusted, this one for example should pass these props:
https://github.com/auth0/docs-v2/blob/3188f5fc37ab01d039dc0eadf0a84cf71da3a91e/auth4genai/snippets/get-started/langchain-next-js/call-others-api.mdx?plain=1#L5-L11

Lets double check them..

Reply:
Unfortunately these will also be conditional right now depending on framework selection, as our langchain-FastAPI / python example is using FCRT / embedded (precompiled) agent flow still.
https://github.com/auth0/docs-v2/blob/3188f5fc37ab01d039dc0eadf0a84cf71da3a91e/auth4genai/snippets/get-started/langchain-fastapi-py/call-others-api.mdx?plain=1#L5-L12

So i suppose suggestion here would be to keep the contents here and below within the  LangChainNextjsCallOthersApi / FastApiCallOthersApi components respectively.


samples today:

https://auth0.com/ai/docs/get-started/call-others-apis-on-users-behalf#langgraph-js-%2B-next-js

https://auth0.com/ai/docs/get-started/call-others-apis-on-users-behalf#langgraph-%2B-fastapi